client/components/oauth/oauth-success.tsx

```diff
@@ -25,7 +25,7 @@ const OAuthSuccess = ({ history }: Props) => {
   if (res.fetching) {
     return <>Loading GraphQL...</>;
   } else if (res.error) {
-    return <>GraphQL Error</>;
+    return <>{history.push('/')}</>;
```

Comment:
This would redirect for all errors. Would it be possible to differentiate a token failure from any other failure?